xbmc/cores/VideoPlayer/VideoRenderers/HwDecRender/RendererVAAPI.h

```diff
@@ -22,9 +22,16 @@
 
 #include "system.h"
 
+#ifdef HAS_GL
 #include "cores/VideoPlayer/VideoRenderers/LinuxRendererGL.h"
+using CRendererVAAPIBase = CLinuxRendererGL;
```

Comment:
or convert `CLinuxRendererGL` and `CLinuxRendererGLES` to just be `CLinuxRenderer` and decide at build time which one would be used.